### ISSUE TITLE
rsa package issue 

### ISSUE THREAD

**Issue by rtommy** [dave-msk/merak#18]
I tried to use the tool with the rsa python package just to see how it works.

The compilation was successful however:

1. It would be valuable if the result could be used as a single pyd file (e.g. `rsa.pyd`) instead of `__init__.pyd` under `rsa` folder (not sure if this is feasible due to module vs. package concept).
2. When I tried to use the pyd file, I got an error (I tried to read in a public key with it)
    ```
    C:\...>merak cythonize rsa out
    C:\...>python
    >>> import rsa
    >>> rsa.__path__
    ['C:\\...\\out\\rsa']
    >>> public_key = rsa.PublicKey.load_pkcs1(publicKey)
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "rsa\\___key.py", line 125, in rsa.___key.AbstractKey.load_pkcs1
      File "rsa\\___key.py", line 330, in rsa.___key.PublicKey._load_pkcs1_pem
    AttributeError: module 'rsa' has no attribute '_PublicKey___pem'
    ```
    where line 330 of `key.py` is:
    `       der = rsa.pem.load_pem(keyfile, "RSA PUBLIC KEY")`

**Comment by dave-msk** [comment]
The original thought when I first developed the tool was indeed to output a single binary file. However, Python interpreters handle packages differently from a module. While a module is defined by a Python source file (`.py`), a package is defined by the directory with an `__init__` file as it is designed with a directory structure in mind. Therefore, for the resulting binary to be able to act as a package, a folder with an `__init__` is inevitable

**Comment by rtommy** [comment]
have you tried to compile the `rsa` package?

**Comment by dave-msk** [comment]
I'm still working on the package restructuring fix, not yet

**Comment by dave-msk** [comment]
This is fixed together with #16 . Please upgrade to 0.3.3.1 and try again

**Comment by rtommy** [comment]
tested with 0.3.3.1, it works.
